src/redux/books/books.js

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+
+const BooksPage = () => (
+  <>
+    <div className="books-container d-flex">
+      <h2>Book List</h2>
+      <button type="button">Remove</button>
+    </div>
+    <input type="text" name="addBooks" placeholder="Book Title" />
+    <button type="button">Add Book</button>
```

Comment:
- [ ] As it is a requirement for this project(`see screenshot below`) and best practice, making sure building blocks of your app are re-usable components, please kindly add a `CreateNewBook` component that; 
- Renders a form with inputs for the title and the author of the book
- Has a button to submit( the button wouldn't add the book yet, this is just a placeholder for now )
![re-usable1](https://user-images.githubusercontent.com/65852621/149915636-548b33f1-b549-42e0-936a-e4d10767f390.png)
![re-usable2](https://user-images.githubusercontent.com/65852621/149915674-fc40340e-d757-428a-87b8-8ab0b365c25a.png)